lib/sles4sap_publiccloud.pm

```diff
@@ -1240,9 +1240,12 @@ sub pacemaker_version {
 
 sub saphanasr_showAttr_version {
     my ($self) = @_;
-    my $version_cmd = 'SAPHanaSR-showAttr --version';
 
-    my $version_output = $self->run_cmd(cmd => $version_cmd, quiet => 1);
+    my $which_output = $self->run_cmd(cmd => 'which SAPHanaSR-showAttr', quiet => 1);
+    record_info('which SAPHanaSR-showAttr', $which_output);
+
+    my $version_output = $self->run_cmd(cmd => "rpm -qf $which_output", quiet => 1);
+    record_info('SAPHanaSR VERSION', $version_output);
 
     if ($version_output =~ /(\d+\.\d+(?:\.\d+)*)/) {
```

Comment:
Is this regexp ok?

`'hello-world-1.2.3.4-12345.6.78.9.end'` --> `1.2.3.4`

Is this part of the version string enough for the purpose of https://jira.suse.com/browse/TEAM-9930 ? @jankohoutek